omero/conf.py

```diff
@@ -392,8 +392,8 @@ def copy_legacy_redirects(app, exception):
     ]
     if app.builder.name == 'html':
         for html_src_path in redirect_files:
-            target_path = app.outdir + '/' + html_src_path
-            src_path = app.srcdir + '/' + html_src_path
+            target_path = (app.outdir / html_src_path)
```

Comment:
broke job on merge-ci https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-docs/221/

Reply:
I'll revert [da299f5](https://github.com/ome/omero-documentation/pull/2346/commits/da299f54c42ccc0d8edf903d343091e37791dc79). I originally thought this was the reason for the readthedocs CI failures.

Note the CI job is using Sphinx 7.1.x but the changes were introduced in [Sphinx 7.2.0](https://www.sphinx-doc.org/en/master/changes.html#release-7-2-0-released-aug-17-2023). So I suspect this codepath will need to be upgraded eventually alongside the version in OME CI. Is there a mechanism to use the same version as readhtedocs?

Reply:
see https://github.com/ome/omero-documentation/pull/2347